app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -71,9 +99,29 @@ export default Controller.extend(ModalFunctionality, {
     return REMINDER_TYPES;
   },
 
+  @discourseComputed()
```

Comment:
you can make this a one liner with the and computed property

http://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/and